docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: "3"
+services:
+    core:
+        image: "${DHIS2_CORE_IMAGE}"
+        labels:
+            - "com.eyeseetea.image-name=${DHIS2_DATA_IMAGE}"
+        volumes:
+            - ./config/DHIS2_home/dhis.conf:/DHIS2_home/dhis.conf
+            - ./config/server.xml:/usr/local/tomcat/conf/server.xml
+            - home:/DHIS2_home
+            - ./config/dhis2-core-start.sh:/usr/local/bin/dhis2-core-start.sh
+            - data:/DHIS2_home/data
+            - "${SCRIPTS_DIR}:/DHIS2_home/scripts"
+        environment:
+            CATALINA_OPTS: "-Dcontext.path="
```

Comment:
I would consider increasing the RAM available for the core to avoid constant crashes.

In my server I have 16GB of RAM and with ```JAVA_OPTS: "-Xmx7500m -Xms4000m"``` I can have 4 dockerized instances without problems. Maybe these values are too high for general purpose but the default ones ARE normally too low for DHIS2.

Reply:
Sound good!